autopts/pybtp/btp/gap.py

```diff
@@ -1240,3 +1289,111 @@ def check_scan_rep_and_rsp(report, response):
         if report in eir and response in eir:
             return True
     return False
+
+
+def gap_padv_configure(include_tx_power, intvl_min, intvl_max):
+    logging.debug("%s", gap_padv_configure.__name__)
+
+    iutctl = get_iut()
+    data_ba = bytearray(struct.pack("<BHH", include_tx_power,
+                                    intvl_min, intvl_max))
+
+    iutctl.btp_socket.send(*GAP['padv_configure'], data=data_ba)
+
+    tuple_data = gap_command_rsp_succ(defs.GAP_PADV_CONFIGURE)
+    __gap_current_settings_update(tuple_data)
+
+
+def gap_padv_start():
+    logging.debug("%s", gap_padv_start.__name__)
+
+    iutctl = get_iut()
+
+    iutctl.btp_socket.send(*GAP['padv_start'])
+
+    tuple_data = gap_command_rsp_succ(defs.GAP_PADV_START)
+    __gap_current_settings_update(tuple_data)
```

Comment:
According to btp_gap.txt the Flags command parameter is missing:
```suggestion
def gap_padv_start(flags):
    logging.debug("%s", gap_padv_start.__name__)

    iutctl = get_iut()
    data_ba = bytearray(struct.pack('<B', flags))

    iutctl.btp_socket.send(*GAP['padv_start'], data=data_ba)

    tuple_data = gap_command_rsp_succ(defs.GAP_PADV_START)
    __gap_current_settings_update(tuple_data)
```

Reply:
Good catch :) added default value of 0 because it's unused for now